### DISCUSSION TITLE
Repositories added to favorites are not showing up in favorites

### DISCUSSION THREAD

**Discussion by jmeridth** [community/community#113687] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.155.0 (126411063)
Device: iPhone 14 Pro (iOS 17.4)
</details>

When I click ellipses on repo screen and choose to add to favorites the repo will not show on my Home Screen favorites. Even going back to repo screen it shows it has been added to my favorites. 

Has been added from repo screen:

![image](https://github.com/community/community/assets/35014/d25fb41c-3bdb-4b95-8b95-d3cb15153909)

Still not showing up in my favorites:

![image](https://github.com/community/community/assets/35014/9e387f66-4208-4289-bcc0-9a61c596bc71)

Search for the repo in favorites and there is a green plus sign that does actually work:

![image](https://github.com/community/community/assets/35014/fb8d2780-092f-447b-8d27-df90fdc14a36)

**Comment by esahmed21** [comment, marked as answer]
Thank you @jmeridth for brining this to our attention.

I'll promptly create an internal ticket for our team to improve this behaviour. To unblock you now, Pulling to refresh in the Home screen should make the repository appear in the favourite list. Thank you for sharing your feedback!

**Reply by jmeridth** [reply]
Pulling to refresh did not list the repository in favorites.

**Reply by esahmed21** [reply]
Hi @jmeridth We were looking into this issue but we were not able to reproduce, do you mind sharing some more information that can help us figure it out? does it happen with every repository or specific repositories? Would updating the app to latest version helps by any mean? Thanks in advance.

**Reply by jmeridth** [reply]
@esahmed21 I will get you more details. 

I'm a beta tester so I'm definitely on latest version of the app. Maybe that is the issue, the was a bug in latest. Not in stable. I'll let you know.

**Reply by jmeridth** [reply]
@esahmed21 this is not happening anymore in latest. It may have never happened in stable. 

Thank you. I'm closing this.
